Annotato/Annotato/Persistence/PersistenceManager/RemotePersistenceManager/RemoteAnnotationsPersistence.swift

```diff
@@ -30,7 +30,12 @@ struct RemoteAnnotationsPersistence: AnnotationsPersistence {
 
         WebSocketManager.shared.send(message: webSocketMessage)
 
-        // We do not get any reponse for the sender from the websocket
+        // We do not get any response for the sender from the websocket
+        return nil
+    }
+
+    func createOrUpdateAnnotationsForLocal(annotations: [Annotation]) -> [Annotation]? {
+        fatalError("RemoteAnnotationsPersistence::createOrUpdateAnnotationForLocal: This function should not be called")
```

Comment:
Remember to rename any strings when you rename the function
```suggestion
        fatalError("RemoteAnnotationsPersistence::createOrUpdateAnnotations: This function should not be called")
```